.circleci/config.yml

```diff
@@ -12,7 +12,7 @@ commands:
   setup_kind:
     description: "Install kind"
     steps:
-      - run: GO111MODULE="on" go get sigs.k8s.io/kind@v0.10.0
+      - run: GO111MODULE="on" go get sigs.k8s.io/kind@latest
```

Comment:
It's ok to bump, but I'd rather pin to a specific kind version to have reliable CI and be deliberate on when we bump.

Reply:
Have there been occasions in the past where a released version negatively impacted MetalLB CI?

Following the latest kind version would allow us to more easily flag deprecated/removed Kubernetes APIs (among other possible breakages). We can still accomplish the same with pinned versions, but requires us to bump it every now and then. I'm fine with such a compromise.

Reply:
There was on MetalLB CI in the past :P, so we can't tell. 
In my opinion the problem with latest is that (and with no pinning to specific versions in general) is that, in the unfortunate case something goes wrong with kind (and not necessarily with the compatibility between us and k8s), our CI will start breaking (or flaking) without us understanding why and needing to jump in firefighting mode.